### DISCUSSION TITLE
Schedule appointment from calendar view

### DISCUSSION THREAD

**Discussion by Maltimore** [nvim-orgmode/orgmode#399] [Ideas]
The date picker has no information about what times I'm busy, so in order to schedule something I have to first go to the agenda view. Once I found an available slot, I have to open an org file or a capture window and create the appointment. It would be cool to be able create an appointment directly from the agenda-calendar view. With this discussion I'd like to gauge whether this would be interesting for others as well.

**Comment by jgollenz** [comment]
I've had that situation. Could you elaborate a bit on the process that you have in mind? 

>create an appointment directly from the agenda-calendar view

That's what you do, I'd like to know how you want to do it, step by step :slightly_smiling_face:

**Reply by Maltimore** [reply]
Ok good to know that there's more interest then :)
I'll think about how exactly this would look like and get back to you.

**Comment by Maltimore** [comment]
I've thought about this a bit more, and will write a detailed proposal soon.
FYI, I described this use-case on the org-mode subreddit and so far nothing interesting there, seems like emacs org-mode doesn't have an obvious way to do this either. https://www.reddit.com/r/orgmode/comments/10ls98b/find_a_time_when_im_free_and_schedule_an_event/
